### ISSUE TITLE
initial sorting not working every time

### ISSUE THREAD

**Issue by thegitfather** [soufianesakhi/feedly-filtering-and-sorting#16]
I'm using the script through Chrome's Tampermonkey extension. Seems like there is a timing issue and the list of my feeds is sometime not sorted - when I open the settings dialog and uncheck/check "Sorting enabled" it works perfectly.

Btw great plugin! Thanks for sharing!

**Comment by soufianesakhi** [comment]
Hello, I am unable to reproduce the issue. Can you send me your subscriptions url and settings (either from console log or by exporting to JSON) ? Do you see any errors in the console ? thanks

**Comment by thegitfather** [comment]
Sure:

```
Feedly page fully loaded
Feedly-Page-FFnS.js:274 TypeError: Cannot read property 'stream' of undefined
    at Feedly-Page-FFnS.js:9:37
    at Feedly-Page-FFnS.js:60:7
    at injectScriptText (eval at E_c (:3:114), <anonymous>:309:19)
    at executeWindow (eval at E_c (:3:114), <anonymous>:270:5)
    at new FeedlyPage (eval at E_c (:3:114), <anonymous>:1284:9)
    at UIManager.eval (eval at E_c (:3:114), <anonymous>:1659:26)
    at AsyncResult.then (eval at E_c (:3:114), <anonymous>:381:23)
    at HTMLSpanElement.eval (eval at E_c (:3:114), <anonymous>:2296:26)
    at eval (eval at E_c (:3:114), <anonymous>:45:30)
    at Array.forEach (native)
VM625:686 Loaded saved subscription: {"filteringEnabled":false,"restrictingEnabled":false,"sortingEnabled":true,"openAndMarkAsRead":true,"markAsReadAboveBelow":false,"markAsReadAboveBelowRead":true,"hideWhenMarkAboveBelow":false,"hideAfterRead":false,"replaceHiddenWithGap":false,"sortingType":0,"advancedControlsReceivedPeriod":{"maxHours":6,"keepUnread":false,"hide":false,"showIfHot":false,"minPopularity":200,"markAsReadVisible":false},"pinHotToTop":false,"additionalSortingTypes":[],"filteringListsByType":{"0":[],"1":[]},"keywordMatchingAreas":[0],"alwaysUseDefaultMatchingAreas":true,"keywordMatchingMethod":0,"url":"---global settings---"}
```
Feed's url are e.g.: `https://feedly.com/i/subscription/feed/http://www.tagesschau.de/xml/rss2` or `https://feedly.com/i/subscription/feed/http://www.heise.de/tp/news-atom.xml`

Another thing I noticed: a max value for `Auto load all unread articles ` would be great, because some feeds have maaany unread ;)

**Comment by soufianesakhi** [comment]
Can try install the script here: https://github.com/soufianesakhi/feedly-filtering-and-sorting/raw/GH-16/script/Feedly_filtering_and_sorting.user.js ?

**Comment by soufianesakhi** [comment]
Can try the script here: https://github.com/soufianesakhi/feedly-filtering-and-sorting/raw/GH-16/script/Feedly_filtering_and_sorting.user.js ?

**Comment by thegitfather** [comment]
Yep, seems to work properly now and the `TypeError` is gone! Thanks and have a good week!

**Comment by soufianesakhi** [comment]
Fixed in v2.7.1
